src/services/Functional/Utilities.ts

```diff
@@ -19,7 +19,13 @@ export let animationFrameRequest: number = 0;
  * @param {Array} colors - Colors for the confetti
  */
 export function throwConfetti(
-  confettiHandler: typeof Object,
+  confettiHandler: (arg0: {
```

Comment:
what is the reference for this?

Reply:
i had attached a screenshot in previous comments and also if you remove this and go for help , then this format is recommended.